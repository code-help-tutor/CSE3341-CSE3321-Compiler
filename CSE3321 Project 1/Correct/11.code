while not (x+y = = == z10 * 101 begin test; test : = test1 - test2;  end