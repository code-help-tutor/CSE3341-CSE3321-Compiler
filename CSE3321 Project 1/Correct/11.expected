WHILE
NOT
LPAREN
ID[x]
ADD
ID[y]
EQUAL
EQUAL
EQUAL
EQUAL
ID[z10]
MULTIPLY
CONST[101]
BEGIN
ID[test]
SEMICOLON
ID[test]
COLON
EQUAL
ID[test1]
SUBTRACT
ID[test2]
SEMICOLON
END
